charts/infra/ff-proxy/README.md

```diff
@@ -0,0 +1,41 @@
+# ff-proxy Helm Chart
```

Comment:
This `README` looks like it's just a copy of the one from `charts/ff-proxy`. Should we write our own README here for this chart and explicitly call out it's for internal use and redirect any customers that find it to `charts/ff-proxy`

Reply:
Agreed